src/config.rs

```diff
@@ -530,6 +619,43 @@ fn maybe_expand_home_dir(path: String) -> Result<String, Error> {
 mod tests {
 
     impl Config {
+        //Method for default testing
+        pub fn default_test() -> Self {
+            use crate::time::SystemTimeProvider;
+            use std::sync::Arc;
```

Comment:
Up at top of module plz

Reply:
Same comment as above - agree this one should go up top.